modules/serverless-service-mesh-enable-sidecar-injection.adoc

```diff
@@ -32,9 +37,8 @@ spec:
 <1> Add the `sidecar.istio.io/inject="true"` annotation.
 <2> Optional: Add the `sidecar.istio.io/rewriteAppHTTPProbers="true"` annotation if you have enabled JSON Web Token (JWT) authentication.
 
-. Apply the Service resource YAML file:
+. Apply the `Service` resource YAML file:
```

Comment:
This might be a topic of discussion for a future team meeting, but I'll share that one alternative might be "Start the service with the sidecar injection:" just to avoid the blandness of "apply" since applying YAML isn't really the customer's goal.

Reply:
Yeah, I'll leave them as apply for now but it would be nice if there's some way to avoid the monotony!